src/styles/blocks/nav.scss

```
@@ -0,0 +1,55 @@
+.nav {
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin: 0 5px;
+    padding: 0 5px;
```

over-engineered a bit
```suggestion
    margin-left: 20px;
```

---

I added paddings to increase the clickable zone, but now I see it was not necessary.